test/system/helpers.bash

```diff
@@ -383,6 +383,13 @@ function journald_unavailable() {
     return 1
 }
 
+# Returns the name of the local pause image.
+function pause_image() {
+    run_podman version --format "{{.Server.Version}}-{{.Server.Built}}" > /dev/null
```

Comment:
Without, it doesn't work as `run_podman` does some magic.  I copied it from another function above.

Reply:
That seems really astonishing. Since it goes against almost everything I know about BATS, could you add a comment explaining why it's needed? (With a little more detail than "doesn't work"?) That is just really unexpected.

Reply:
`git blame` brought me to one of your commit 9fd7ab50f82c7:
>  This function is intended to be used as '$(podman_runtime)', i.e.
>  our caller wants our output. run_podman() messes with output because
> it emits the command invocation to stdout, hence the redirection.